lib/dbaccessor_test.go

```diff
@@ -287,14 +287,63 @@ func testInsertAndGetUser(ta TestAccessor, t *testing.T) {
 func testInsertAndGetFilteredUsers(ta TestAccessor, t *testing.T) {
 	t.Log("TestInsertAndGetFilteredUsers")
 	ta.Truncate()
+	// create user
```

Comment:
This test seems to have a lot of duplicate code. I think structuring this with common setup, subtests (or a table driven structure), and a simple function iterate over rows with a filter function would DRY this up and make it easier to read and manage.

Basically, I see insert users from a struct literal (where the attributes don't seem to change but type and affiliation do), insert affiliations (0 or 1 right now), call get filtered users, assert the result set contains the expected data.

Reply:
I have removed the duplicate code.